eks-without-vpc/init-k8s.tf

```diff
@@ -0,0 +1,54 @@
+
+resource "kubernetes_config_map" "aws-auth" {
+  metadata {
+    name      = "aws-auth"
+    namespace = "kube-system"
+  }
+
+  data {
+    mapRoles = <<ROLES
+- groups:
+  - system:bootstrappers
+  - system:nodes
+  rolearn: ${aws_iam_role.aws-eks-node.arn}
+  username: system:node:{{EC2PrivateDNSName}}
+ROLES
+    mapUsers = "${var.master-users}"
+  }
+
+    depends_on = ["aws_autoscaling_group.aws-eks"]
+
+}
+
+resource "kubernetes_service_account" "spinnaker-sa" {
```

Comment:
Can you please remove all resources related with service accounts, secrets, role binding, etc like it was done for `eks` scripts?